### PR TITLE
Migrate to poi

### PR DESCRIPTION
**Pull request by vladimyr**
This PR swaps webpack with poi. By doing so we are getting rid of tons of explicit webpack configuration settings :wink:

### CODE REVIEW

**Comment by vladimyr** [comment]
@kronicker Do not merge it yet, need to remove `locus` from deps list :running_man: :man_facepalming:

File: app.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ server.register([Inert, db, boomErrors, routes, monitor], (err) => {
   server.start(function (err) {
     if (err) { throw err; }
 
-    console.log(`Server running at : ${server.info.uri}`);
+    console.log(`Server running at: ${server.info.uri}`);
```
Comment (vladimyr) [inline app.js:25]: That space simply triggered my OCD :man_facepalming:

File: poi.config.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+};
+
+module.exports = options => ({
+  entry: [
+    require.resolve('bootstrap-loader'),
+    'client/main.js'
+  ],
+  preset: [
+    require('poi-preset-buble')(),
+    require('poi-preset-eslint')({ mode: '*' })
```
Comment (vladimyr) [inline poi.config.js:18]: Buble preset has turned `dangerousForOf` transform on by default.
Using wildcard here will include `eslint-loader` in both development and production mode.

**Comment by vladimyr** [comment]
Ready for merge :airplane:

File: app.js (hunk 1 above)
Comment (kronicker) [reply app.js:25]: :laughing: :+1:

File: poi.config.js (hunk 2 above)
Comment (kronicker) [reply poi.config.js:18]: Great, appreciate the clarification :wink:

**Review by kronicker — APPROVED** [review]
Me likey! :100: :fireworks:

**Pull request merged**